scripts/install.sh

```diff
@@ -56,6 +57,7 @@ OPTIONS:
    -d      Directory where kubent will be installed. Default is /usr/local/bin
    -a      Architecture to install (x86_64 only atm.). Default is to auto-detect.
    -o      OS (linux, macos). Default is to auto-detect.
+   -n      Install latest nightly, if set to 1. Default is 0
```

Comment:
Can we make this boolean switch, instead of an argument with a value? (same as the -h)

I.e.:
```bash
./install.sh -n ...
```

as opposed to
```bash
./install.sh -n1 ...
```